docs/user-guide.md

```diff
@@ -1109,6 +1109,11 @@ pipeline for more details in the setup and integration.
 - `TARGET_ACCOUNTS`: comma separated list of target accounts.
 - `TARGET_OUS`: comma separated list of target leaf OUs (parent
   OUs are supported).
+- `TARGET_TAGS`: comma separated list of tag=value couples. All accounts
+  matching at least one of the tags will be targeted by the pipeline.
+  
+  **Eg:** `TARGET_TAGS=environment=prd,cost-center=ccoe` will match all
+  accounts tagged with `environment = prd` OR `cost-center = ccoe`
```

Comment:
Tag keys and values are also allowed to have a `=` key. See: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-resource-tags.html

It looks like a comma is not allowed, so that should be good.
Is there another way to pass the target_tags?

Reply:
Updated the documentation with the new filter format, see 217e71736fd930b56957024face541f78227cc67